projects/packages/my-jetpack/_inc/components/upsell-banner/jetpack-manage-banner/index.jsx

```diff
@@ -0,0 +1,48 @@
+import { getRedirectUrl } from '@automattic/jetpack-components';
+import { __ } from '@wordpress/i18n';
+import UpsellBanner from '../';
+import jetpackManageIcon from './jetpack-manage.svg';
+
+/**
+ * Jetpack Manager Banner component that renders a banner with CTAs.
+ *
+ * @param {object} props - Component props.
+ * @returns {object} The JetpackManageBanner component.
+ */
+const JetpackManageBanner = props => {
+	const { isAgencyAccount } = props;
+
+	// Set up the first CTA
+	const cta1Label = __( 'Learn more', 'jetpack-my-jetpack' );
```

Comment:
I think it might be clearer to distinguish cta1Label and cta2Label a bit more (and the other similar variables such as cta1WithIcon, cta1URL, etc), because at a quick glance the two blend in with each other and are harder to distinguish. Maybe something like ctaLearnMore and ctaSignUpFree? What do you think?

Reply:
Yep, it's a better name! Thank you!

Reply:
Updated here: https://github.com/Automattic/jetpack/pull/35078/commits/9bb34a683b2fc08bd45889c0da61b21a6e9f54ca